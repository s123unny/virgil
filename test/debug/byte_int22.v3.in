b ../core/byte_int22.v3 3
b ../core/byte_int22.v3 4
b ../core/byte_int22.v3 5
b ../core/byte_int22.v3 6
b ../core/byte_int22.v3 7
b ../core/byte_int22.v3 8
b ../core/byte_int22.v3 9
b ../core/byte_int22.v3 10
b ../core/byte_int22.v3 14
run
bt
info l
c
bt
info l
c
q
