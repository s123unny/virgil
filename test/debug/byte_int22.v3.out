breakpoint|../core/byte_int22.v3 3|0
breakpoint|../core/byte_int22.v3 4|1
breakpoint|../core/byte_int22.v3 5|2
breakpoint|../core/byte_int22.v3 6|3
breakpoint|../core/byte_int22.v3 7|4
breakpoint|../core/byte_int22.v3 8|5
breakpoint|../core/byte_int22.v3 9|6
breakpoint|../core/byte_int22.v3 10|7
breakpoint|../core/byte_int22.v3 14|8
stopOnBreakpoint
bt|main|../core/byte_int22.v3|3
variable|0|arg|0|int|false
stopOnBreakpoint
bt|x|../core/byte_int22.v3|14
bt|main|../core/byte_int22.v3|3
variable|0|i|-1|int|false
variable|1|a|0|byte|false
end
result|255
